tests/vmipreset_test.go

```diff
@@ -309,6 +324,9 @@ var _ = Describe("[rfe_id:609][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 			getPreset(virtClient, memoryPrefix)
```

Comment:
can we get rid of the "getPreset()" function here now?

Reply:
fixed